test/280_with_or_without_dns_test.sh

```diff
@@ -12,7 +12,9 @@ check_dns() {
     chk=$1
     shift
 
-    container=$(weave_on $HOST1 run $@ $IP/24 -t --add-host=$STATIC:$STATIC_IP $DNS_IMAGE /bin/sh)
+    container=$(docker_on $HOST1 run $(dns_args $HOST1 "$@") -dt $DNS_IMAGE /bin/sh)
+    weave_on $HOST1 attach $IP/24 "$@" --rewrite-hosts --add-host=$STATIC:$STATIC_IP $container
+    echo $chk $HOST1 $container $TARGET
```

Comment:
Given there is `$chk $HOST1 $container $TARGET` on line 18, is this:
a) just a temporary add which was accidentally checked in?
b) something we want to conditionally print, e.g. `[ -z "$DEBUG" ] || echo ...`? or
c) something we want to always print from now onwards?